bundles/org.openhab.ui/web/src/assets/definitions/blockly/blocks-persistence.js

```diff
@@ -16,18 +16,21 @@ export default function defineOHBlocks_Persistence (f7, isGraalJs) {
       this.appendDummyInput()
         .appendField('get the')
         .appendField(new Blockly.FieldDropdown([
-          ['average', 'averageSince'], ['delta', 'deltaSince'],
-          ['deviation', 'deviationSince'], ['variance', 'varianceSince'], ['evolution rate', 'evolutionRate'],
-          ['minimum', 'minimumSince'], ['maximum', 'maximumSince'], ['sum', 'sumSince'],
+          ['state average', 'averageSince'], ['state delta', 'deltaSince'],
+          ['state deviation', 'deviationSince'], ['state variance', 'varianceSince'], ['evolution rate', 'evolutionRateSince'],
+          ['state minimum', 'minimumSince'], ['state maximum', 'maximumSince'], ['state sum', 'sumSince'],
```

Comment:
IMO average state, minimum state, maximum sound better.

Reply:
This is the same like before only that I have moved it into the dropdown (it was part of the label before the dropdown). IMHO, it is actually the state average not the average state. Also it is consistent to the others.